File: SourceCode Others Languagens/c++/soma.cpp
#include <iostream>
#include <locale.h>

using namespace std;

int main(void){
  setlocale(LC_ALL, "Portuguese");
  double n1, n2, avarage;

  cout<< "Digite número 1: "; cin>> n1;
  cout<< "Digite número 2: "; cin>> n2;

  avarage = ( n1 + n2 ) / 2.0;
  cout<< "\nResultado = " << avarage << endl;
}

File: SourceCode Others Languagens/java/soma.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
    double n1, n2, avarage;

    Scanner sc = new Scanner(System.in);

    System.out.print("Digite Número 1: "); n1 = sc.nextDouble();
    System.out.print("Digite Número 2: "); n2 = sc.nextDouble();

    avarage = ( n1 + n2 ) / 2.0;

    System.out.println("Resultado = " + avarage);
    sc.close();
    }
}
